Powershell scripts/Custom Secure Score/CustomSecureScore.psm1

```diff
@@ -156,9 +156,9 @@ Function Update-AzSecurityCenterSecureScoreControlMappings {
 
     $metadata = $PolicySetDefinition.Properties.Metadata
     $securityCenter = Get-Property -Object $metadata -PropertyName "securityCenter" -CreateIfNotExist -DefaultCreationValue $(New-Object -TypeName psobject)
-    $securityCenterEnabled = Get-Property -Object $securityCenter -PropertyName "enabled" -CreateIfNotExist -DefaultCreationValue $true
+    #$securityCenterEnabled = Get-Property -Object $securityCenter -PropertyName "enabled" -CreateIfNotExist -DefaultCreationValue $true
     $secureScore = Get-Property -Object $securityCenter -PropertyName "secureScore" -CreateIfNotExist -DefaultCreationValue $(New-Object -TypeName psobject)
-    $existingControlMappings = Get-Property -Object $secureScore -PropertyName "controlMappings" -CreateIfNotExist -DefaultCreationValue @()
+    #$existingControlMappings = Get-Property -Object $secureScore -PropertyName "controlMappings" -CreateIfNotExist -DefaultCreationValue @()

```

Comment:
Uncomment these back - see that they create the property if it does not exist.
If the result is unused, pipe the result to Out-Null